### DISCUSSION TITLE
Shelly uni e citofono

### DISCUSSION THREAD

**Discussion by dirtymagic17** [indomus/forum#1436] [Hardware & Firmware]
Buongiorno a tutti,
vorrei domotizzare un citofono comelit basic 2603u con SHELLY UNI per notifica suonata e apertura cancello.
Qualcuno mi può aiutare con i collegamenti ? Ho letto su altro forum che per apertura cancello servè un relè open collector... qualcuno ha già esperienze?

**Comment by RobyBS** [comment]
.. interessante .. seguo .. 👍
